src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -1297,7 +1297,7 @@ public String takeScreenshot(String baseName) {
         if (t != null) {
             if (t instanceof ScreenshotException) {
                 String encodedScreenshot = ((ScreenshotException)t).getBase64EncodedScreenshot();
-                result = new Base64Encoder().decode(encodedScreenshot);
+                result = new Base64().decode(encodedScreenshot);
```

Comment:
Let's not add a separate dependency to do base64 decode. Can't we use https://docs.oracle.com/javase/8/docs/api/java/util/Base64.Decoder.html?